src/main/java/module_two_1/Runner.java

```diff
@@ -0,0 +1,76 @@
+package module_two_1;
+
+import module_two_1.exceptions.DataEntryExceptions;
+import module_two_1.models.Faculty;
+import module_two_1.models.Group;
+import module_two_1.models.Student;
+import module_two_1.models.University;
+import module_two_1.models.enums.DisciplineNames;
+import module_two_1.models.enums.FacultiesNames;
+import module_two_1.models.enums.GroupNames;
+import module_two_1.models.enums.Messenger;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+public class Runner {
+    private static final Logger LOGGER = Logger.getLogger(Runner.class.getName());
+    private static final String INPUT_FILE = "src/in.txt";
+    private static final int SKIP5 = 5;
+    private static final int SKIP10 = 10;
+    private static final int SKIP15 = 15;
+    private static final int SKIP20 = 20;
+    private static final int LIMIT5 = 5;
+    private static final int INDEX_0 = 0;
+    private static final int INDEX_3 = 3;
+
+    public static void main(String[] args) {
+        List<Faculty> facultyList;
+        Student someStudent;
+        try {
+            facultyList = getPreconditions();
+            University university = new University("GSU", facultyList);
+            System.out.println(university);
+            someStudent = facultyList.get(INDEX_0).getGroups().get(INDEX_0).getStudents().get(INDEX_3);
+            System.out.println(String.format("%s %s = %.2f", Messenger.COUNT_BY_STUDENT, someStudent.getName(),
```

Comment:
I see this string is duplicated, so it may be stored as a constant instead of hardcoding

Reply:
Added to constants